client/src/api/tasks.js

```diff
@@ -0,0 +1,18 @@
+export async function getTask(taskId) {
+  const response = await fetch(`/tasks/${taskId}`, { method: 'GET' });
+  if (!response.ok) {
+    throw new Error(response.statusText);
+  }
+  const task = await response.json();
+
+  return task;
+}
+export async function getAllTasks() {
+  const response = await fetch(`/tasks/`, { method: 'GET' });
+  if (!response.ok) {
+    throw new Error(response.statusText);
+  }
+  const tasks = await response.json();
+
+  return tasks;
+}
```

Comment:
I think this needs to get a day as parameter so it can fetch tasks for a specific day

Reply:
Yes thats the plan for the next step after the modal